packages/web-components/src/components/va-language-toggle/test/va-language-toggle.e2e.ts

```diff
@@ -2,41 +2,37 @@ import { newE2EPage } from "@stencil/core/testing";
 import { axeCheck } from "../../../testing/test-helpers";
 
 describe('va-language-toggle', () => {
-  const enHref = "/resources/the-pact-act-and-your-va-benefits/";
-  const esHref = "/resources/the-pact-act-and-your-va-benefits-esp/"
-  const tlHref = "/resources/the-pact-act-and-your-va-benefits-tl/"
-
   it('renders', async () => {
     const page = await newE2EPage();
-    await page.setContent(`<va-language-toggle en-href="${enHref}" es-href="${esHref}" tl-href="${tlHref}" />`);
+    await page.setContent(`<va-language-toggle en-href="#" es-href="#" tl-href="#" />`);
```

Comment:
@it-harrison Is it necessary to have specific `href` links for these tests? Unfortunately, I'm getting a lot of warnings because of them when I run `yarn test`. Would it be ok to just mock the href value with `#` instead? The tests are still passing.

![Screenshot 2024-11-19 at 11 21 48 AM](https://github.com/user-attachments/assets/eb4e0409-95e3-4130-b5eb-45fe46b42aa8)

![Screenshot 2024-11-19 at 11 21 44 AM](https://github.com/user-attachments/assets/053e918f-48f7-428f-86f9-085dd6d8d979)